src/app/pages/login/login.page.scss

```diff
@@ -37,16 +37,24 @@
        margin: 0px 16px 0px 16px;
        padding-bottom: 20px;
     }    
- 
+
     .submit-btn {
         --padding-start: 10%;
         --padding-end: 10%;
-        --padding-top: 10%;
-        --padding-bottom: 10%;
+        --padding-top: 24px;
```

Comment:
Using pixels is never a good choice, they are screen size dependent. Can you try to find another alternative?